src/pages/Referral/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { colors } from '@hedviginsurance/brand'
+import {
+  TranslationsConsumer,
+  TranslationsPlaceholderConsumer,
+} from '@hedviginsurance/textkeyfy'
+import { Button } from 'components/buttons'
+import { Page } from 'components/utils/Page'
+import * as React from 'react'
+import styled from 'react-emotion'
+import { RouteComponentProps } from 'react-router'
+import { MobileContext } from 'utils/mobileContext'
+import { getFirebaseLinkDomain } from './util'
+
+type ReferralProps = RouteComponentProps<{
+  code: string
+}>
+
+const PageWrapper = styled('div')({
+  ['@media (min-width: 700px)']: {
```

Comment:
no need for square brackets here, does prettier not remove this?

Reply:
Nope 🤔 But I'll remove it!

Reply:
haha good